packages/enum-converter/src/parsers/python/python.parser.ts

```diff
@@ -25,45 +16,51 @@ export class PythonParser extends FileParser {
     const entries: EnumEntry[] = [];
     // scan ast for enums
     walk(ast, {
-      enter: (node: Node, parent: Node) => {
-        if (
-          node &&
-          node.type === 'FunctionDeclaration' &&
-          node.body &&
-          node.body.body.length > 1 &&
-          this.isEnumCallExpression(node.body.body[0])
-        ) {
-          const entryName = node?.id?.name;
-          const entryValues: EnumValue[] = this.extractEnumValues(node);
-          if (entryName) {
-            const enumEntry = new EnumEntry(entryName, entryValues);
-            entries.push(enumEntry);
-          }
+      enter: (node: BaseNode , parent: BaseNode ) => {
+        if (node.type === "FunctionDeclaration") {
+          let functionDeclaration: FunctionDeclaration = <FunctionDeclaration>node ;
```

Comment:
Use TypeScript type inference. Also you don't need to cast since estree types using discriminated union